apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Tests/RetryTest.cs

```diff
@@ -30,23 +29,30 @@ namespace Google.Cloud.Storage.V1.Tests;
 /// </summary>
 public class RetryTest
 {
-    /// <summary>
-    /// A set of retry options using default timing, but a predicate that retries on 404 (NotFound) only.
-    /// </summary>
-    private static readonly RetryOptions s_customPredicateOptions = new RetryOptions(RetryTiming.Default, RetryPredicate.FromErrorCodes(404));
+    private static readonly RetryTiming s_customRetryTiming = new(TimeSpan.FromSeconds(1), TimeSpan.FromSeconds(5), 2);
+    private static readonly RetryPredicate s_customPredicate = RetryPredicate.FromErrorCodes(404, 400);
+    private static readonly RetryOptions s_customRetryOptions = new(s_customRetryTiming, s_customPredicate);
```

Comment:
Given that we only use the fields above when constructing this, I'd just rewrite this as:
```csharp
private static readonly RetryOptions s_customRetryOptions = new RetryOptions(
    new RetryTiming(TimeSpan.FromSeconds(1), TimeSpan.FromSeconds(5), 2),
    RetryPredicate.FromErrorCodes(404, 400)
);
```

(I'd explicitly name the types rather than using target-typed `new` just for clarity, as you'd be calling two different constructors.)